server/upgrade.js

```diff
@@ -0,0 +1,71 @@
+const mongo   = require("promised-mongo")
+const db      = mongo("mongodb://localhost/bydesign", ["authors", "posts", "metadata", "categories"])
+const logger  = require("beautiful-log")
+const config  = require("../config")
+const utils   = require("./utils")
+
+let isLessThan = (desired, upgrader) => 
+  version => version <= desired ? upgrader() : version
```

Comment:
Also, thoughts on `eslint` rule `no-confusing-arrow`? It would require you to surround the ternary with parens as well (since `=>` is visually also a greater-than-or-equal which can be confusing)